### ISSUE TITLE
Filtering UI issues

### ISSUE THREAD

**Issue by Nothing4You** [jellyfin/jellyfin-web#2429]
**Describe The Bug**
I'm trying to filter my library to only display HD movies.
SD also gets checked, I'm not sure if it actually also does include SD content.
SD filtering checkbox does not stay checked and does not show filtered movie count.

**Steps To Reproduce**
1. Go to movie library
2. Click on filter icon
3. Filter for HD
4. Click background to return to library view
5. Click on filter icon
6. HD and SD are selected
1. Start fresh without filters
2. Filter for SD
3. SD movies are found, movie count isn't shown

**Expected Behavior**
Filtering for HD should not include nor check SD.
SD filter should show a movie count.

**Logs**
The following shows up in the browser console, not sure if that's related.
```
inputManager: tried to process command with no action assigned: unknown
```

**Screenshots**
1. Initial unfiltered library view
![image](https://user-images.githubusercontent.com/3694534/108481386-6e890100-7298-11eb-85f2-d2e6533c2be1.png)
2. Select HD filter
![image](https://user-images.githubusercontent.com/3694534/108481440-81033a80-7298-11eb-9b95-54267ba20ce8.png)
3. Click in background and return to library view. Some filter has been applied as fewer items are listed now
![image](https://user-images.githubusercontent.com/3694534/108481490-911b1a00-7298-11eb-9905-73d85fa31f95.png)
4. Open filter again, now SD is also checked
![image](https://user-images.githubusercontent.com/3694534/108481531-9bd5af00-7298-11eb-8e26-723547407140.png)
1. Start fresh without filter
2. Select SD filter, now I see SD movies but no longer a movie count. Also, when returning to filters SD is unchecked.
![image](https://user-images.githubusercontent.com/3694534/108482813-400c2580-729a-11eb-9e16-afe5d373ed95.png)

**System (please complete the following information):**
 - Platform: macOS
 - Browser: Firefox
 - Jellyfin Version: 10.6.4

**Additional information:**
Initial request: `https://jellyfin/Users/<userid>/Items?SortBy=DateCreated%2CSortName%2CProductionYear&SortOrder=Descending&IncludeItemTypes=Movie&Recursive=true&Fields=PrimaryImageAspectRatio%2CMediaSourceCount%2CBasicSyncInfo&ImageTypeLimit=1&EnableImageTypes=Primary%2CBackdrop%2CBanner%2CThumb&StartIndex=0&ParentId=<libraryid>&Limit=100`
Request after setting HD filter: `https://jellyfin/Users/<userid>/Items?SortBy=DateCreated%2CSortName%2CProductionYear&SortOrder=Descending&IncludeItemTypes=Movie&Recursive=true&Fields=PrimaryImageAspectRatio%2CMediaSourceCount%2CBasicSyncInfo&ImageTypeLimit=1&EnableImageTypes=Primary%2CBackdrop%2CBanner%2CThumb&StartIndex=0&ParentId=<libraryid>&Limit=100&IsHD=true`
Filtering for SD after having no filter: `https://jellyfin/Users/<userid>/Items?SortBy=DateCreated%2CSortName%2CProductionYear&SortOrder=Descending&IncludeItemTypes=Movie&Recursive=true&Fields=PrimaryImageAspectRatio%2CMediaSourceCount%2CBasicSyncInfo&ImageTypeLimit=1&EnableImageTypes=Primary%2CBackdrop%2CBanner%2CThumb&StartIndex=0&ParentId=<libraryid>&Limit=100&IsHD=false`

**Comment by Nothing4You** [comment]
shut up stalebot

**Comment by Nothing4You** [comment]
*insert useless comment here that generates activity for the bot to see*

**Comment by agrudsky** [comment]
I am having a similar issue on 10.8.0-beta1. Filtering by resolution (HD, 4K, etc) doesn't work, but filtering by watched status works. If I select to only show 4K media, all media is still shown.

I have tried on multiple browsers to no avail.

**Comment by Nothing4You** [comment]
I just didn't want to bother with generating useless activity for stalebot anymore.
Unfortunately there does not seem to be any human review, just stalebot closing issues.

I suggest raising a new issue, as I can't reopen this one.
Be prepared to just talk to stalebot for a year until you lose interest.

**Comment by thornbill** [comment]
Confirming that issues are still relevant is not useless activity. You may not like stalebot but it would be impossible for our very small team to triage the hundreds of issues we receive without it cleaning up inactive issues. Yes there are people who spend countless hours reviewing issues and contributing changes to Jellyfin. We need the community to help us manage issues though and I really don't think that is too much to ask.

As for this particular issue, I had looked at it in the past but could not reproduce it. I will attempt to reproduce it again today.

**Comment by Nothing4You** [comment]
I can understand the lack of resources for triaging, but I'd appreciate if there was a way to mark issues as confirmed, or at least provide some feedback that more info is required.
As of right now I can still reproduce this in my library on 10.7.7.
Admittedly, I have not tested on a recent pre-release, but at least on the latest stable release the issue still exists.

From my point of view this ticket was created a little over a year ago and so far my only communication was with stalebot.
As I can see from reactions and @agrudsky's comment there seem to be at least 4 other users facing the same issue.

I'd be happy to add more information or test things if that helps.

**Comment by Nothing4You** [comment]
Seems to be fixed in 10.8.0 via #3577, works fine now.
Thank you @thornbill 

For the second issue mentioned above where item counts aren't shown, that doesn't appear to be related to the filter.
When there's less than 100 items (therefore no pagination required), there's no item count shown.

**Comment by agrudsky** [comment]
I am still having this issue on 10.8.1 (Docker). No Video Type filters work -- SD, HD, 4K, etc. They all just show my entire library without actually filtering anything. Any ideas? @thornbill
